### ISSUE TITLE
Slow Planning Times on Cold Connection

### ISSUE THREAD

**Issue by jbylund** [timescale/timescaledb#425]
I'm seeing that the first query on a new connection is much slower than expected to plan.  This effect is lessened on repeating an identical query and gone by the third execution of the query.  Changing query parameters seems to not disrupt this "caching" behavior.  Example:

```
third_party=# EXPLAIN (analyze, buffers, verbose) SELECT level0top, level1, ad_server_imps, ad_server_clicks, ad_server_ctr, viewable_imps, not_viewable_imps, not_measurable_imps, viewable_imps_rate, eligible_imps, measurable_imps_rate, ad_server_cpm_and_cpc_revenue FROM third_party.ad_server_stats WHERE (active) AND ((ad_date >= '2018-01-31'::date)) AND ((ad_date <= '2018-01-31'::date)) AND ((level0top = 'TEST'::text));
                                                                                                                                                                                                                        QUERY PLAN                                                                                                                                                                                                                         
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 Append  (cost=0.00..10348.29 rows=8541 width=105) (actual time=0.173..104.663 rows=35800 loops=1)
   Buffers: shared hit=15662
   ->  Seq Scan on third_party.ad_server_stats  (cost=0.00..0.00 rows=1 width=1112) (actual time=0.016..0.016 rows=0 loops=1)
         Output: ad_server_stats.level0top, ad_server_stats.level1, ad_server_stats.ad_server_imps, ad_server_stats.ad_server_clicks, ad_server_stats.ad_server_ctr, ad_server_stats.viewable_imps, ad_server_stats.not_viewable_imps, ad_server_stats.not_measurable_imps, ad_server_stats.viewable_imps_rate, ad_server_stats.eligible_imps, ad_server_stats.measurable_imps_rate, ad_server_stats.ad_server_cpm_and_cpc_revenue
         Filter: (ad_server_stats.active AND (ad_server_stats.ad_date >= '2018-01-31'::date) AND (ad_server_stats.ad_date <= '2018-01-31'::date) AND ((ad_server_stats.level0top)::text = 'TEST'::text))
   ->  Index Scan using _hyper_1_47_chunk_third_party_l0_slicer_date_w_active_idx on _timescaledb_internal._hyper_1_47_chunk  (cost=0.56..1216.18 rows=977 width=106) (actual time=0.064..0.064 rows=0 loops=1)
         Output: _hyper_1_47_chunk.level0top, _hyper_1_47_chunk.level1, _hyper_1_47_chunk.ad_server_imps, _hyper_1_47_chunk.ad_server_clicks, _hyper_1_47_chunk.ad_server_ctr, _hyper_1_47_chunk.viewable_imps, _hyper_1_47_chunk.not_viewable_imps, _hyper_1_47_chunk.not_measurable_imps, _hyper_1_47_chunk.viewable_imps_rate, _hyper_1_47_chunk.eligible_imps, _hyper_1_47_chunk.measurable_imps_rate, _hyper_1_47_chunk.ad_server_cpm_and_cpc_revenue
         Index Cond: (((_hyper_1_47_chunk.level0top)::text = 'TEST'::text) AND (_hyper_1_47_chunk.ad_date >= '2018-01-31'::date) AND (_hyper_1_47_chunk.ad_date <= '2018-01-31'::date))
         Buffers: shared hit=4
   ->  Index Scan using _hyper_1_48_chunk_third_party_l0_slicer_date_w_active_idx on _timescaledb_internal._hyper_1_48_chunk  (cost=0.56..9132.11 rows=7563 width=105) (actual time=0.091..78.133 rows=35800 loops=1)
         Output: _hyper_1_48_chunk.level0top, _hyper_1_48_chunk.level1, _hyper_1_48_chunk.ad_server_imps, _hyper_1_48_chunk.ad_server_clicks, _hyper_1_48_chunk.ad_server_ctr, _hyper_1_48_chunk.viewable_imps, _hyper_1_48_chunk.not_viewable_imps, _hyper_1_48_chunk.not_measurable_imps, _hyper_1_48_chunk.viewable_imps_rate, _hyper_1_48_chunk.eligible_imps, _hyper_1_48_chunk.measurable_imps_rate, _hyper_1_48_chunk.ad_server_cpm_and_cpc_revenue
         Index Cond: (((_hyper_1_48_chunk.level0top)::text = 'TEST'::text) AND (_hyper_1_48_chunk.ad_date >= '2018-01-31'::date) AND (_hyper_1_48_chunk.ad_date <= '2018-01-31'::date))
         Buffers: shared hit=15658
 Planning time: 585.748 ms
 Execution time: 117.746 ms
(15 rows)

Time: 708.383 ms
third_party=# EXPLAIN (analyze, buffers, verbose) SELECT level0top, level1, ad_server_imps, ad_server_clicks, ad_server_ctr, viewable_imps, not_viewable_imps, not_measurable_imps, viewable_imps_rate, eligible_imps, measurable_imps_rate, ad_server_cpm_and_cpc_revenue FROM third_party.ad_server_stats WHERE (active) AND ((ad_date >= '2018-01-31'::date)) AND ((ad_date <= '2018-01-31'::date)) AND ((level0top = 'TEST'::text));
                                                                                                                                                                                                                        QUERY PLAN                                                                                                                                                                                                                         
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 Append  (cost=0.00..10348.30 rows=8541 width=105) (actual time=0.093..114.704 rows=35800 loops=1)
   Buffers: shared hit=15662
   ->  Seq Scan on third_party.ad_server_stats  (cost=0.00..0.00 rows=1 width=1112) (actual time=0.005..0.005 rows=0 loops=1)
         Output: ad_server_stats.level0top, ad_server_stats.level1, ad_server_stats.ad_server_imps, ad_server_stats.ad_server_clicks, ad_server_stats.ad_server_ctr, ad_server_stats.viewable_imps, ad_server_stats.not_viewable_imps, ad_server_stats.not_measurable_imps, ad_server_stats.viewable_imps_rate, ad_server_stats.eligible_imps, ad_server_stats.measurable_imps_rate, ad_server_stats.ad_server_cpm_and_cpc_revenue
         Filter: (ad_server_stats.active AND (ad_server_stats.ad_date >= '2018-01-31'::date) AND (ad_server_stats.ad_date <= '2018-01-31'::date) AND ((ad_server_stats.level0top)::text = 'TEST'::text))
   ->  Index Scan using _hyper_1_47_chunk_third_party_l0_slicer_date_w_active_idx on _timescaledb_internal._hyper_1_47_chunk  (cost=0.56..1216.18 rows=977 width=106) (actual time=0.026..0.026 rows=0 loops=1)
         Output: _hyper_1_47_chunk.level0top, _hyper_1_47_chunk.level1, _hyper_1_47_chunk.ad_server_imps, _hyper_1_47_chunk.ad_server_clicks, _hyper_1_47_chunk.ad_server_ctr, _hyper_1_47_chunk.viewable_imps, _hyper_1_47_chunk.not_viewable_imps, _hyper_1_47_chunk.not_measurable_imps, _hyper_1_47_chunk.viewable_imps_rate, _hyper_1_47_chunk.eligible_imps, _hyper_1_47_chunk.measurable_imps_rate, _hyper_1_47_chunk.ad_server_cpm_and_cpc_revenue
         Index Cond: (((_hyper_1_47_chunk.level0top)::text = 'TEST'::text) AND (_hyper_1_47_chunk.ad_date >= '2018-01-31'::date) AND (_hyper_1_47_chunk.ad_date <= '2018-01-31'::date))
         Buffers: shared hit=4
   ->  Index Scan using _hyper_1_48_chunk_third_party_l0_slicer_date_w_active_idx on _timescaledb_internal._hyper_1_48_chunk  (cost=0.56..9132.12 rows=7563 width=105) (actual time=0.058..76.099 rows=35800 loops=1)
         Output: _hyper_1_48_chunk.level0top, _hyper_1_48_chunk.level1, _hyper_1_48_chunk.ad_server_imps, _hyper_1_48_chunk.ad_server_clicks, _hyper_1_48_chunk.ad_server_ctr, _hyper_1_48_chunk.viewable_imps, _hyper_1_48_chunk.not_viewable_imps, _hyper_1_48_chunk.not_measurable_imps, _hyper_1_48_chunk.viewable_imps_rate, _hyper_1_48_chunk.eligible_imps, _hyper_1_48_chunk.measurable_imps_rate, _hyper_1_48_chunk.ad_server_cpm_and_cpc_revenue
         Index Cond: (((_hyper_1_48_chunk.level0top)::text = 'TEST'::text) AND (_hyper_1_48_chunk.ad_date >= '2018-01-31'::date) AND (_hyper_1_48_chunk.ad_date <= '2018-01-31'::date))
         Buffers: shared hit=15658
 Planning time: 195.398 ms
 Execution time: 133.768 ms
(15 rows)

Time: 330.406 ms
third_party=# EXPLAIN (analyze, buffers, verbose) SELECT level0top, level1, ad_server_imps, ad_server_clicks, ad_server_ctr, viewable_imps, not_viewable_imps, not_measurable_imps, viewable_imps_rate, eligible_imps, measurable_imps_rate, ad_server_cpm_and_cpc_revenue FROM third_party.ad_server_stats WHERE (active) AND ((ad_date >= '2018-01-31'::date)) AND ((ad_date <= '2018-01-31'::date)) AND ((level0top = 'TEST'::text));
                                                                                                                                                                                                                        QUERY PLAN                                                                                                                                                                                                                         
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 Append  (cost=0.00..10350.33 rows=8543 width=105) (actual time=0.103..121.058 rows=35800 loops=1)
   Buffers: shared hit=15662
   ->  Seq Scan on third_party.ad_server_stats  (cost=0.00..0.00 rows=1 width=1112) (actual time=0.005..0.005 rows=0 loops=1)
         Output: ad_server_stats.level0top, ad_server_stats.level1, ad_server_stats.ad_server_imps, ad_server_stats.ad_server_clicks, ad_server_stats.ad_server_ctr, ad_server_stats.viewable_imps, ad_server_stats.not_viewable_imps, ad_server_stats.not_measurable_imps, ad_server_stats.viewable_imps_rate, ad_server_stats.eligible_imps, ad_server_stats.measurable_imps_rate, ad_server_stats.ad_server_cpm_and_cpc_revenue
         Filter: (ad_server_stats.active AND (ad_server_stats.ad_date >= '2018-01-31'::date) AND (ad_server_stats.ad_date <= '2018-01-31'::date) AND ((ad_server_stats.level0top)::text = 'TEST'::text))
   ->  Index Scan using _hyper_1_47_chunk_third_party_l0_slicer_date_w_active_idx on _timescaledb_internal._hyper_1_47_chunk  (cost=0.56..1217.19 rows=978 width=106) (actual time=0.023..0.023 rows=0 loops=1)
         Output: _hyper_1_47_chunk.level0top, _hyper_1_47_chunk.level1, _hyper_1_47_chunk.ad_server_imps, _hyper_1_47_chunk.ad_server_clicks, _hyper_1_47_chunk.ad_server_ctr, _hyper_1_47_chunk.viewable_imps, _hyper_1_47_chunk.not_viewable_imps, _hyper_1_47_chunk.not_measurable_imps, _hyper_1_47_chunk.viewable_imps_rate, _hyper_1_47_chunk.eligible_imps, _hyper_1_47_chunk.measurable_imps_rate, _hyper_1_47_chunk.ad_server_cpm_and_cpc_revenue
         Index Cond: (((_hyper_1_47_chunk.level0top)::text = 'TEST'::text) AND (_hyper_1_47_chunk.ad_date >= '2018-01-31'::date) AND (_hyper_1_47_chunk.ad_date <= '2018-01-31'::date))
         Buffers: shared hit=4
   ->  Index Scan using _hyper_1_48_chunk_third_party_l0_slicer_date_w_active_idx on _timescaledb_internal._hyper_1_48_chunk  (cost=0.56..9133.14 rows=7564 width=105) (actual time=0.071..79.831 rows=35800 loops=1)
         Output: _hyper_1_48_chunk.level0top, _hyper_1_48_chunk.level1, _hyper_1_48_chunk.ad_server_imps, _hyper_1_48_chunk.ad_server_clicks, _hyper_1_48_chunk.ad_server_ctr, _hyper_1_48_chunk.viewable_imps, _hyper_1_48_chunk.not_viewable_imps, _hyper_1_48_chunk.not_measurable_imps, _hyper_1_48_chunk.viewable_imps_rate, _hyper_1_48_chunk.eligible_imps, _hyper_1_48_chunk.measurable_imps_rate, _hyper_1_48_chunk.ad_server_cpm_and_cpc_revenue
         Index Cond: (((_hyper_1_48_chunk.level0top)::text = 'TEST'::text) AND (_hyper_1_48_chunk.ad_date >= '2018-01-31'::date) AND (_hyper_1_48_chunk.ad_date <= '2018-01-31'::date))
         Buffers: shared hit=15658
 Planning time: 4.596 ms
 Execution time: 141.415 ms
(15 rows)
```

Postgres version 10.2 from pgdg ppa.
Timescale version 0.8.0 compiled locally.

**Comment by jbylund** [comment]
I believe this was immediately following a `vacuum full analyze` of pg_catalog.pg_{attribute,statistic} and a couple of timescale the table currently has 50 chunks.  Even more extreme and possibly interesting:

```
third_party=# explain select sum(1) from third_party.ad_server_stats;
                                                   QUERY PLAN                                                    
-----------------------------------------------------------------------------------------------------------------
 Finalize Aggregate  (cost=458559154.48..458559154.49 rows=1 width=8)
   ->  Gather  (cost=458559153.55..458559154.46 rows=9 width=8)
         Workers Planned: 9
         ->  Partial Aggregate  (cost=458558153.55..458558153.56 rows=1 width=8)
               ->  Append  (cost=0.00..452878956.44 rows=2271678843 width=0)
                     ->  Parallel Seq Scan on ad_server_stats  (cost=0.00..0.00 rows=1 width=0)
                     ->  Parallel Seq Scan on _hyper_1_1_chunk  (cost=0.00..102.55 rows=3955 width=0)
                     ->  Parallel Seq Scan on _hyper_1_2_chunk  (cost=0.00..102.27 rows=3927 width=0)
                     ->  Parallel Seq Scan on _hyper_1_3_chunk  (cost=0.00..71336.77 rows=894477 width=0)
                     ->  Parallel Seq Scan on _hyper_1_4_chunk  (cost=0.00..57168.67 rows=712067 width=0)
                     ->  Parallel Seq Scan on _hyper_1_5_chunk  (cost=0.00..532600.42 rows=4645242 width=0)
                     ->  Parallel Seq Scan on _hyper_1_6_chunk  (cost=0.00..662301.39 rows=5661139 width=0)
                     ->  Parallel Seq Scan on _hyper_1_7_chunk  (cost=0.00..2037909.34 rows=15341234 width=0)
                     ->  Parallel Seq Scan on _hyper_1_8_chunk  (cost=0.00..1210677.59 rows=9100259 width=0)
                     ->  Parallel Seq Scan on _hyper_1_9_chunk  (cost=0.00..2638271.89 rows=16202889 width=0)
                     ->  Parallel Seq Scan on _hyper_1_10_chunk  (cost=0.00..3666487.54 rows=22217654 width=0)
                     ->  Parallel Seq Scan on _hyper_1_11_chunk  (cost=0.00..5156046.11 rows=32520511 width=0)
                     ->  Parallel Seq Scan on _hyper_1_12_chunk  (cost=0.00..4887448.92 rows=30504792 width=0)
                     ->  Parallel Seq Scan on _hyper_1_13_chunk  (cost=0.00..13067021.57 rows=73210057 width=0)
                     ->  Parallel Seq Scan on _hyper_1_14_chunk  (cost=0.00..12875303.12 rows=68964112 width=0)
                     ->  Parallel Seq Scan on _hyper_1_15_chunk  (cost=0.00..9806124.37 rows=53859437 width=0)
                     ->  Parallel Seq Scan on _hyper_1_16_chunk  (cost=0.00..13668840.67 rows=75107567 width=0)
                     ->  Parallel Seq Scan on _hyper_1_17_chunk  (cost=0.00..14248589.25 rows=77013425 width=0)
                     ->  Parallel Seq Scan on _hyper_1_18_chunk  (cost=0.00..19357660.57 rows=80012457 width=0)
                     ->  Parallel Seq Scan on _hyper_1_19_chunk  (cost=0.00..20786950.68 rows=104505068 width=0)
                     ->  Parallel Seq Scan on _hyper_1_20_chunk  (cost=0.00..15866739.91 rows=84746791 width=0)
                     ->  Parallel Seq Scan on _hyper_1_21_chunk  (cost=0.00..6666950.23 rows=41310923 width=0)
                     ->  Parallel Seq Scan on _hyper_1_22_chunk  (cost=0.00..5575454.69 rows=34562069 width=0)
                     ->  Parallel Seq Scan on _hyper_1_23_chunk  (cost=0.00..7947891.22 rows=41748922 width=0)
                     ->  Parallel Seq Scan on _hyper_1_24_chunk  (cost=0.00..6333293.88 rows=35202988 width=0)
                     ->  Parallel Seq Scan on _hyper_1_25_chunk  (cost=0.00..5628262.55 rows=34618455 width=0)
                     ->  Parallel Seq Scan on _hyper_1_26_chunk  (cost=0.00..4725768.59 rows=29709459 width=0)
                     ->  Parallel Seq Scan on _hyper_1_27_chunk  (cost=0.00..6646158.93 rows=37052793 width=0)
                     ->  Parallel Seq Scan on _hyper_1_28_chunk  (cost=0.00..4439712.86 rows=22926386 width=0)
                     ->  Parallel Seq Scan on _hyper_1_29_chunk  (cost=0.00..12203369.93 rows=64900093 width=0)
                     ->  Parallel Seq Scan on _hyper_1_30_chunk  (cost=0.00..10896586.81 rows=57400581 width=0)
                     ->  Parallel Seq Scan on _hyper_1_31_chunk  (cost=0.00..10179160.06 rows=55835406 width=0)
                     ->  Parallel Seq Scan on _hyper_1_32_chunk  (cost=0.00..8767761.65 rows=47434165 width=0)
                     ->  Parallel Seq Scan on _hyper_1_33_chunk  (cost=0.00..13703331.60 rows=56824460 width=0)
                     ->  Parallel Seq Scan on _hyper_1_34_chunk  (cost=0.00..12629876.13 rows=61387113 width=0)
                     ->  Parallel Seq Scan on _hyper_1_35_chunk  (cost=0.00..16363595.16 rows=67798616 width=0)
                     ->  Parallel Seq Scan on _hyper_1_36_chunk  (cost=0.00..16237689.21 rows=78053521 width=0)
                     ->  Parallel Seq Scan on _hyper_1_37_chunk  (cost=0.00..16515263.97 rows=70418297 width=0)
                     ->  Parallel Seq Scan on _hyper_1_38_chunk  (cost=0.00..16668731.01 rows=86283801 width=0)
                     ->  Parallel Seq Scan on _hyper_1_39_chunk  (cost=0.00..16285482.15 rows=82819215 width=0)
                     ->  Parallel Seq Scan on _hyper_1_40_chunk  (cost=0.00..15064360.38 rows=51828638 width=0)
                     ->  Parallel Seq Scan on _hyper_1_41_chunk  (cost=0.00..17492433.64 rows=77383064 width=0)
                     ->  Parallel Seq Scan on _hyper_1_42_chunk  (cost=0.00..18862858.03 rows=88058203 width=0)
                     ->  Parallel Seq Scan on _hyper_1_43_chunk  (cost=0.00..17675699.59 rows=73652259 width=0)
                     ->  Parallel Seq Scan on _hyper_1_44_chunk  (cost=0.00..19426133.78 rows=83162478 width=0)
                     ->  Parallel Seq Scan on _hyper_1_45_chunk  (cost=0.00..3368404.16 rows=20292416 width=0)
                     ->  Parallel Seq Scan on _hyper_1_46_chunk  (cost=0.00..3540938.35 rows=16166035 width=0)
                     ->  Parallel Seq Scan on _hyper_1_47_chunk  (cost=0.00..8855952.42 rows=46652442 width=0)
                     ->  Parallel Seq Scan on _hyper_1_48_chunk  (cost=0.00..9444657.14 rows=51428114 width=0)
                     ->  Parallel Seq Scan on _hyper_1_49_chunk  (cost=0.00..70308.82 rows=802182 width=0)
                     ->  Parallel Seq Scan on _hyper_1_50_chunk  (cost=0.00..65185.89 rows=738688 width=0)
(56 rows)

Time: 30329.083 ms (00:30.329)

third_party=# explain select sum(1) from third_party.ad_server_stats;
                                                   QUERY PLAN                                                    
-----------------------------------------------------------------------------------------------------------------
 Finalize Aggregate  (cost=458559154.48..458559154.49 rows=1 width=8)
   ->  Gather  (cost=458559153.55..458559154.46 rows=9 width=8)
         Workers Planned: 9
         ->  Partial Aggregate  (cost=458558153.55..458558153.56 rows=1 width=8)
               ->  Append  (cost=0.00..452878956.44 rows=2271678843 width=0)
                     ->  Parallel Seq Scan on ad_server_stats  (cost=0.00..0.00 rows=1 width=0)
                     ->  Parallel Seq Scan on _hyper_1_1_chunk  (cost=0.00..102.55 rows=3955 width=0)
                     ->  Parallel Seq Scan on _hyper_1_2_chunk  (cost=0.00..102.27 rows=3927 width=0)
                     ->  Parallel Seq Scan on _hyper_1_3_chunk  (cost=0.00..71336.77 rows=894477 width=0)
                     ->  Parallel Seq Scan on _hyper_1_4_chunk  (cost=0.00..57168.67 rows=712067 width=0)
                     ->  Parallel Seq Scan on _hyper_1_5_chunk  (cost=0.00..532600.42 rows=4645242 width=0)
                     ->  Parallel Seq Scan on _hyper_1_6_chunk  (cost=0.00..662301.39 rows=5661139 width=0)
                     ->  Parallel Seq Scan on _hyper_1_7_chunk  (cost=0.00..2037909.34 rows=15341234 width=0)
                     ->  Parallel Seq Scan on _hyper_1_8_chunk  (cost=0.00..1210677.59 rows=9100259 width=0)
                     ->  Parallel Seq Scan on _hyper_1_9_chunk  (cost=0.00..2638271.89 rows=16202889 width=0)
                     ->  Parallel Seq Scan on _hyper_1_10_chunk  (cost=0.00..3666487.54 rows=22217654 width=0)
                     ->  Parallel Seq Scan on _hyper_1_11_chunk  (cost=0.00..5156046.11 rows=32520511 width=0)
                     ->  Parallel Seq Scan on _hyper_1_12_chunk  (cost=0.00..4887448.92 rows=30504792 width=0)
                     ->  Parallel Seq Scan on _hyper_1_13_chunk  (cost=0.00..13067021.57 rows=73210057 width=0)
                     ->  Parallel Seq Scan on _hyper_1_14_chunk  (cost=0.00..12875303.12 rows=68964112 width=0)
                     ->  Parallel Seq Scan on _hyper_1_15_chunk  (cost=0.00..9806124.37 rows=53859437 width=0)
                     ->  Parallel Seq Scan on _hyper_1_16_chunk  (cost=0.00..13668840.67 rows=75107567 width=0)
                     ->  Parallel Seq Scan on _hyper_1_17_chunk  (cost=0.00..14248589.25 rows=77013425 width=0)
                     ->  Parallel Seq Scan on _hyper_1_18_chunk  (cost=0.00..19357660.57 rows=80012457 width=0)
                     ->  Parallel Seq Scan on _hyper_1_19_chunk  (cost=0.00..20786950.68 rows=104505068 width=0)
                     ->  Parallel Seq Scan on _hyper_1_20_chunk  (cost=0.00..15866739.91 rows=84746791 width=0)
                     ->  Parallel Seq Scan on _hyper_1_21_chunk  (cost=0.00..6666950.23 rows=41310923 width=0)
                     ->  Parallel Seq Scan on _hyper_1_22_chunk  (cost=0.00..5575454.69 rows=34562069 width=0)
                     ->  Parallel Seq Scan on _hyper_1_23_chunk  (cost=0.00..7947891.22 rows=41748922 width=0)
                     ->  Parallel Seq Scan on _hyper_1_24_chunk  (cost=0.00..6333293.88 rows=35202988 width=0)
                     ->  Parallel Seq Scan on _hyper_1_25_chunk  (cost=0.00..5628262.55 rows=34618455 width=0)
                     ->  Parallel Seq Scan on _hyper_1_26_chunk  (cost=0.00..4725768.59 rows=29709459 width=0)
                     ->  Parallel Seq Scan on _hyper_1_27_chunk  (cost=0.00..6646158.93 rows=37052793 width=0)
                     ->  Parallel Seq Scan on _hyper_1_28_chunk  (cost=0.00..4439712.86 rows=22926386 width=0)
                     ->  Parallel Seq Scan on _hyper_1_29_chunk  (cost=0.00..12203369.93 rows=64900093 width=0)
                     ->  Parallel Seq Scan on _hyper_1_30_chunk  (cost=0.00..10896586.81 rows=57400581 width=0)
                     ->  Parallel Seq Scan on _hyper_1_31_chunk  (cost=0.00..10179160.06 rows=55835406 width=0)
                     ->  Parallel Seq Scan on _hyper_1_32_chunk  (cost=0.00..8767761.65 rows=47434165 width=0)
                     ->  Parallel Seq Scan on _hyper_1_33_chunk  (cost=0.00..13703331.60 rows=56824460 width=0)
                     ->  Parallel Seq Scan on _hyper_1_34_chunk  (cost=0.00..12629876.13 rows=61387113 width=0)
                     ->  Parallel Seq Scan on _hyper_1_35_chunk  (cost=0.00..16363595.16 rows=67798616 width=0)
                     ->  Parallel Seq Scan on _hyper_1_36_chunk  (cost=0.00..16237689.21 rows=78053521 width=0)
                     ->  Parallel Seq Scan on _hyper_1_37_chunk  (cost=0.00..16515263.97 rows=70418297 width=0)
                     ->  Parallel Seq Scan on _hyper_1_38_chunk  (cost=0.00..16668731.01 rows=86283801 width=0)
                     ->  Parallel Seq Scan on _hyper_1_39_chunk  (cost=0.00..16285482.15 rows=82819215 width=0)
                     ->  Parallel Seq Scan on _hyper_1_40_chunk  (cost=0.00..15064360.38 rows=51828638 width=0)
                     ->  Parallel Seq Scan on _hyper_1_41_chunk  (cost=0.00..17492433.64 rows=77383064 width=0)
                     ->  Parallel Seq Scan on _hyper_1_42_chunk  (cost=0.00..18862858.03 rows=88058203 width=0)
                     ->  Parallel Seq Scan on _hyper_1_43_chunk  (cost=0.00..17675699.59 rows=73652259 width=0)
                     ->  Parallel Seq Scan on _hyper_1_44_chunk  (cost=0.00..19426133.78 rows=83162478 width=0)
                     ->  Parallel Seq Scan on _hyper_1_45_chunk  (cost=0.00..3368404.16 rows=20292416 width=0)
                     ->  Parallel Seq Scan on _hyper_1_46_chunk  (cost=0.00..3540938.35 rows=16166035 width=0)
                     ->  Parallel Seq Scan on _hyper_1_47_chunk  (cost=0.00..8855952.42 rows=46652442 width=0)
                     ->  Parallel Seq Scan on _hyper_1_48_chunk  (cost=0.00..9444657.14 rows=51428114 width=0)
                     ->  Parallel Seq Scan on _hyper_1_49_chunk  (cost=0.00..70308.82 rows=802182 width=0)
                     ->  Parallel Seq Scan on _hyper_1_50_chunk  (cost=0.00..65185.89 rows=738688 width=0)
(56 rows)

Time: 8.344 ms
```

**Comment by jbylund** [comment]
This behavior seems to have evaporated without any configuration changes or explicit vacuum analyze activity on my part.  `explain select sum(1) from third_party.ad_server_stats;` now takes ~100ms on a cold connection.

**Comment by davidkohn88** [comment]
This is odd behavior. We've tried to recreate a few times and had little success. The fact that it went away without explanation is also exceedingly odd. Do let us know if you have any more info on it.

**Comment by mfreed** [comment]
Hi @jbylund Anything else to report on this, or should we close out this issue?  0.10 also significantly changed some of the planning behavior with many chunks.

**Comment by jbylund** [comment]
Looks like this has been fixed in 0.10, thanks!

```
explain analyze select min(date) as start_date from sometable where someattr = 'SOMEVALUE';
```

0.9.0
```
 Planning time: 4714.348 ms
 Execution time: 5.927 ms
```

0.10.1
```
 Planning time: 36.142 ms
 Execution time: 67.521 ms
```
